e2e/utils/dom.js

```diff
@@ -15,7 +15,7 @@ async function submitForm(page, formData) {
 async function waitForSubmissionSuccess(page) {
   await page.waitFor(
     SUBMISSIONS_SUCCESS => document.location.href === SUBMISSIONS_SUCCESS,
-    {},
+    { timeout: 60000 },
```

Comment:
we shouldn't increase the timeout, something else is causing this test to fail

Reply:
I removed it already, but it get back during merge :/